xapian_db.gemspec

```
@@ -5,7 +5,7 @@ $:.unshift lib unless $:.include?(lib)
 
 Gem::Specification.new do |s|
   s.name         = %q{xapian_db}
-  s.version      = "1.3.10"
+  s.version      = "1.3.11"
```

This should be a major bump (IMHO) as it changes the sidekiq version requirement:
```suggestion
  s.version      = "2.0.0"
```

---

Hello @gernotkogler, do we update the version? 
Please could you let me know? Thanks.